config/windows-containerd-monitor-filelog.json

```diff
@@ -20,6 +20,11 @@
 			"type": "temporary",
 			"reason": "IncompatibleContainer",
 			"pattern": ".*CreateComputeSystem.*"
+		},
+		{
+			"type": "temporary",
+			"reason": "ContainerProcessStartFailed",
+			"pattern": ".*failed to start.*hcsshim::System::CreateProcess.*"
```

Comment:
Can you provide an example of the message in the description of this PR?
Is it possible to also make this pattern more specific so that its possible to distinguish other issues?